english/investigate-glowing-pants/investigate-glowing-pants.md

```diff
@@ -2,4 +2,5 @@ You take your pants off, turn them upside down, and shake them vigorously.
 
 A small pixie falls out and angrily bites you on the nose before flying off to hide in a corner of the room.
 
-[Ignore the pixie for now](../marshmallow.md)
\ No newline at end of file
+[Ignore the pixie for now](../marshmallow.md)
+[Ask the pixie a few questions](../interrogate-pixie/interrogate-pixie.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.